otx/algorithms/classification/adapters/mmcls/models/classifiers/sam_classifier.py

```diff
@@ -1,12 +1,11 @@
+"""Module for defining SAMClassifier for classification task."""
 # Copyright (C) 2022 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 #
-
-from collections import OrderedDict
+# pylint: disable=too-many-branches, unused-argument, too-many-nested-blocks, too-many-locals, ungrouped-imports
```

Comment:
Please disabling pylint as minimum as possible. How about setting in on necessary functions or classes?